nilearn/group_sparse_covariance.py

```diff
@@ -545,7 +545,7 @@ def fit(self, subjects, y=None):
             _group_sparse_covariance, memory_level=1)(
                 self.covariances_, n_samples, self.alpha,
                 tol=self.tol, max_iter=self.max_iter,
-                verbose=self.verbose - 1, debug=False)
+                verbose=max(0, self.verbose - 1), debug=False)
```

Comment:
> I'd prefer not to--I believe the two overlap a bit, and so I'd need to
> de-merge, then re-merge. If I leave things as they are, if / when #366
> is merged to master, I will be able to rebase much more smoothly (even
> if changes occur there).

Then I suggest that we stop reviewing this PR till #366 is merged.

> In other projects, we dealt with this by looking at the changes on a
> commit-by-commit basis.

That's not our workflow. We like to work based on PR. The github
interface makes this very easy to do. The benefit is that looking at the
whole PR makes it harder to miss something like the introduction of a
bug.